src/Razor/test/VSCode.FunctionalTest/tests/CodeLens.test.ts

```diff
@@ -4,40 +4,23 @@
  * ------------------------------------------------------------------------------------------ */
 
 import * as assert from 'assert';
-import { afterEach, before, beforeEach } from 'mocha';
+import { beforeEach } from 'mocha';
 import * as path from 'path';
 import * as vscode from 'vscode';
 import {
     mvcWithComponentsRoot,
-    pollUntil,
-    waitForProjectReady,
 } from './TestUtil';
 
 let razorPath: string;
 let razorDoc: vscode.TextDocument;
 let razorEditor: vscode.TextEditor;
 
 suite('CodeLens', () => {
-    before(async () => {
-        await waitForProjectReady(mvcWithComponentsRoot);
-    });
-
     beforeEach(async () => {
         razorPath = path.join(mvcWithComponentsRoot, 'Views', 'Shared', 'NavMenu.razor');
         razorDoc = await vscode.workspace.openTextDocument(razorPath);
         razorEditor = await vscode.window.showTextDocument(razorDoc);
-    });
-
-    afterEach(async () => {
-        await vscode.commands.executeCommand('workbench.action.revertAndCloseActiveEditor');
-        await pollUntil(async () => {
-            await vscode.commands.executeCommand('workbench.action.closeAllEditors');
-            if (vscode.window.visibleTextEditors.length === 0) {
-                return true;
-            }
-
-            return false;
-        }, /* timeout */ 3000, /* pollInterval */ 500, true /* suppress timeout */);
+        await new Promise(r => setTimeout(r, 5000));
```

Comment:
This seems super hacky, does it actually pass reliably? Wondering if we should be skipping these

Reply:
I'm on the fence. This test hasn't failed even once after this change. But then again I don't know if I ran it enough times. We can always take the approach of skip and file issue if/when we see these tests fail. I'd prefer to run them when we can on the CI so we get more data.

Reply:
Alright, sounds fine to me 😄